File: help/eng_gb/exercicios_da_ficha_3/Ficha03_022/Ficha03_022.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Angle             : " ;
    $ang = floatval(isset($_POST["ang"])? $_POST["ang"] : isset($_GET["ang"]) ? $_GET["ang"] : 0);
    
    echo $ang . "<br/>";// print input 
    echo  "Hyperbolic sine     = " . ( sinh( $ang )  ) . "\n" ;
    echo  "Hyperbolic Cosine     = " . ( cosh( $ang )  ) . "\n" ;
    echo  "Hyperbolic tangent     = " . ( tanh( $ang )  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Tue, 2022 Sep 13 21:59

?>

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_022/Ficha03_022.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha03_022 {

    public:
        int main() {
            cout << "Angle \t\t\t: ";
            float ang;
            cin >> ang;
            cout << "Hyperbolic sine \t= " <<( sinh( ang )  ) << "\n";
            cout << "Hyperbolic Cosine \t= " <<( cosh( ang )  ) << "\n";
            cout << "Hyperbolic tangent \t= " <<( tanh( ang )  );
            return 0;
        }
};// end of class Ficha03_022
// Execute main function
int main(){
    Ficha03_022 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:59
